packages/repack/src/types.ts

```diff
@@ -82,7 +82,7 @@ export interface StartArguments extends CommonArguments {
 export interface DevServerOptions {
   /** Whether to start development server. */
   enabled?: boolean;
-  /** Hostname under which to run the development server. Defaults to `localhost`. */
+  /** Hostname under which to run the development server. */
```

Comment:
```suggestion
  /**
   * Hostname or IP address under which to run the development server.
   * When left unspecified, it will listen on all available network interfaces, similarly to listening on '0.0.0.0'.
   */
```